### PR TITLE
chore: cleanup spelling, syntax and typos

### PR DESCRIPTION
**Pull request by mc-jones**
Just a PR to cleanup some typos and improve syntax in a few areas. Some of these had been resolved previously, but got dropped somewhere before we formalized our branches/release process.

### CODE REVIEW

File: src/Apps/ArtAdvisor/ArtAdvisorRoutes.tsx [hunk 1]
```diff
@@ -2,28 +2,28 @@ import loadable from "@loadable/component"
 import { AppRouteConfig } from "System/Router/Route"
 
 const ArtAdvisorApp = loadable(
-  () => import(/* webpackChunkName: "jobsBundle" */ "./ArtAdvisorApp"),
+  () => import(/* webpackChunkName: "advisorBundle" */ "./ArtAdvisorApp"),
```
Comment (mc-jones) [inline src/Apps/ArtAdvisor/ArtAdvisorRoutes.tsx:5]: https://github.com/artsy/force/pull/13871#discussion_r1596941508

**Comment by mc-jones** [comment]
> I know its annoying and trivial, but it would be nice if we also followed idiomatic JS naming where we can. (Noticing lots of `snake_case` vars here and there)

I hear you. This was carried over from the openAI docs, which tends to use snake case for function tool names, which we then map to the idiomatic javascript function names. Will push up changes.

**Comment by damassi** [comment]
Figured as much 👍 just trying to get ahead of the point where we start promoting code and / or incorporating. Its definitely no big deal.

**Review by anandaroop — APPROVED** [review]
LGTM, feel free to merge or leave open if you want to push up any casing changes.

**Pull request merged**